api_tests/user_post_success.csv

```diff
@@ -0,0 +1,11 @@
+id,username,firstName,lastName,email,password,phone,userStatus,statusCode
+1,johnDoe,John,Doe,johnDoe@example.com,123456,1234567890,Active,200
```

Comment:
{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request","status":422,"detail":"Your request is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["body","password"],"severity":"Error","code":"type","message":"Request body property password must be string"}]}

Reply:
since artillery uses csv-helper to for natively converting csv feild values.
so password entity converted to number.